tests/Resources/Page/OCPDashboard/Pods/Pods.robot

```diff
@@ -180,4 +180,35 @@ Get User Server Node
     [Arguments]    ${username}=${TEST_USER.USERNAME}
     ${pod_name} =    Get User Notebook Pod Name    ${username}
     ${node_name} =    Run    oc describe Pod ${pod_name} -n rhods-notebooks | grep Node: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"/"); print b[1]}'
-    [Return]    ${node_name}
\ No newline at end of file
+    [Return]    ${node_name}
+
+Get Pod Logs From UI With Container
+    [Documentation]    Get pod logs text from OCP UI
+    [Arguments]    ${namespace}    ${pod_search_term}    ${container_button_id}
+    Navigate To Page    Workloads    Pods
+    Search Last Item Instance By Title In OpenShift Table    search_term=${pod_search_term}
+    ...    namespace=${namespace}
+    Click Link    xpath://tr[@data-key='0-0']/td/span/a
+    Click Link    Logs
+    Sleep    4
+    Click Button    xpath://div[@class="resource-log"]//button[@data-test-id="dropdown-button"]
+    Click Button    id=${container_button_id}
+    Sleep    4
+    Capture Page Screenshot    logs_page.png
+    ${log_lines_flag}=    Run Keyword And Return Status    Wait Until Page Contains Element
+    ...    xpath://div[@class='log-window__lines']
+    ${log_list_flag}=    Run Keyword And Return Status    Wait Until Page Contains Element
+    ...    xpath://div[@class='pf-c-log-viewer__list']
+    IF    ${log_lines_flag} == ${TRUE}
+        ${logs_text}=    Get Text    xpath://div[@class='log-window__lines']
+    ELSE IF    ${log_list_flag} == ${TRUE}
+        Click Link    Raw
+        Switch Window    NEW
+        ${logs_text}=    Get Text    xpath://pre
+        Close Window
+        Switch Window    MAIN
```

Comment:
Can this be made into a keyword?

Reply:
sry 
But it was already written by someone and honestly I don't have idea about this section
is ok if I do not create a new keyword for these lines?

Reply:
Yeah, that's fine

Reply:
@bdattoma and @lugi0 
I have modified ` Get Pod Logs From UI ` to avoid duplications in code.
and it does not affect other codes.